alerter.cpp

```diff
@@ -1,7 +1,16 @@
 #include <iostream>
 #include <assert.h>
 
+#define AlertTest 0 //0 for networkAlertTest and 1 for networkAlertStub
+
 int alertFailureCount = 0;
+int networkAlertTest(float celcius) {
+    std::cout << "ALERT: Temperature is " << celcius << " celcius.\n";
+    // Return 200 for ok
+    // Return 500 for not-ok
+    // stub always succeeds and returns 200
+    return 200;
+}
 
 int networkAlertStub(float celcius) {

```

Comment:
stub and production code is the same file. They should be kept separate so that it helps in future maintainability and also test code can be easily excluded during shipment of software to field.